open-machine-learning-jupyter-book/ml-fundamentals/regression/tools-of-the-trade.md

```diff
@@ -0,0 +1,238 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Tools of the trade
+
+## Get started with Python and Scikit-learn for regression models
+
+![Summary of regressions in a sketchnote](../../../images/ml-regression/ml-regression.png)
+
+> Sketchnote by [Tomomi Imura](https://www.twitter.com/girlie_mac)
+
+## Introduction
+
+In these four sections, you will discover how to build regression models. We will discuss what these are for shortly. But before you do anything, make sure you have the right tools in place to start the process!
+
+In this section, you will learn how to:
+
+- Configure your computer for local machine learning tasks.
+- Work with Jupyter notebooks.
+- Use Scikit-learn, including installation.
+- Explore linear regression with a hands-on exercise.
+
+## Installations and configurations
+
+[![Setup Python with Visual Studio Code](https://img.youtube.com/vi/yyQM70vi7V8/0.jpg)](https://youtu.be/yyQM70vi7V8 "Setup Python with Visual Studio Code")
+
+```{note}
+Click the image above for a video: using Python within VS Code.
+```
+
+1. **Install Python**. Ensure that [Python](https://www.python.org/downloads/) is installed on your computer. You will use Python for many data science and machine learning tasks. Most computer systems already include a Python installation. There are useful [Python Coding Packs](https://code.visualstudio.com/learn/educators/installers?WT.mc_id=academic-77952-leestott) available as well, to ease the setup for some users.
+
+   Some usages of Python, however, require one version of the software, whereas others require a different version. For this reason, it's useful to work within a [virtual environment](https://docs.python.org/3/library/venv.html).
```

Comment:
remove the spaces at the beginning. Or this is necessary to keep the number bullet correct?